oc_mirror.sh

```diff
@@ -34,6 +34,11 @@ function update_docker_config() {
 
 function setup_quay_mirror_registry() {
 
+   if sudo podman container exists registry; then
+     echo "The podman registry is currently running and will cause a conflict with quay registry. Run \"registry_cleanup.sh\" to remove podman registry."
```

Comment:
Is there any specific scenario that will make it happen? I'd expect that a `make clean` could be sufficient in such case

Reply:
Yes the reason this now happens is that registry_cleanup.sh was moved out of `make clean` and into `make realclean` to avoid having to rebuild the registry on each run - https://github.com/openshift-metal3/dev-scripts/pull/1498.  Which works fine until switch registry backends, which should not happen that often.